azavea/ism-watershed-wellness-snapshot
# Integrate prototype

## Overview

Adds the prototype assets, adjusts the DOM to match the prototypes, and adds a few simple components.

Connects #37 

### Demo

![jan-28-2019 13-33-23](https://user-images.githubusercontent.com/1042475/51857881-59cc2000-2301-11e9-8a9d-755a3509ac82.gif)

### Notes

Lots of follow-up issues: #40, #41, #42, #43 

## Testing Instructions

- Run `./scripts/update`, followed by `./scripts/server`.
- Visit the site and verify there are no errors in the console.
